src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -0,0 +1,2394 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents an expression that creates a pointer value by taking the address of a reference.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    internal sealed partial class AddressOfExpression : Operation, IAddressOfExpression
```

Comment:
First thing in our list to discuss https://github.com/dotnet/roslyn/issues/18246